### DISCUSSION TITLE
Geräte Icon als Icon im HomeKitTile verwenden

### DISCUSSION THREAD

**Discussion by simonssoftware** [Zefau/ioBroker.jarvis#1401] [Q&A] [answered]
Servus miteinander,

schaff ich es irgendwie im HomeKitTile anstelle des Eurozeichens (Icon des States "E5Preis") das Icon des Gerätes zu verwenden?

Was ich schon probiert habe ist, dass ich das Icon des States auf das BayWa-Icon gesetzt habe aber dann ist das im Popup auch das Baywa-Icon beim Preis und das möchte ich nicht.

Aktuelles HomeKitTile:
![image](https://user-images.githubusercontent.com/50638485/147661003-945e419f-9031-4bae-84f4-668cc5710b25.png)

Konfiguration:
![image](https://user-images.githubusercontent.com/50638485/147661165-7719bf36-aa4a-4a97-9cf8-68b6cff2c7b6.png)
![image](https://user-images.githubusercontent.com/50638485/147661190-3f5981c6-3682-444c-9ad1-ff1141a56426.png)
![image](https://user-images.githubusercontent.com/50638485/147661506-b4164b05-3617-4f6e-a5e1-09a2b3f66530.png)

Grüß,
Simon

**Comment by Zefau** [comment, marked as answer]
Leider nein. Mach gerne einen Feature Request auf, dann bau ich in 3.1 eine Option ein.

**Reply by simonssoftware** [reply]
Alles klar danke für die schnelle Antwort, ich erstelle einen Feature Request. 👍
